src/Api/State/PersonalDataStateProcessor.php

```diff
@@ -54,6 +55,14 @@ public function process(mixed $data, Operation $operation, array $uriVariables =
             return $this->persistProcessor->process($data, $operation, $uriVariables, $context);
         }
 
+        if ($data instanceof DonneePersonnelle && str_starts_with((string) $context['operation']->getName(), (string) ApiRoutes::TOGGLE_VISIBILITY)) {
+            if (!$user instanceof User) {
+                $data->setBPrive(true);
+            }
```

Comment:
`$data->setBPrive(!$data->getBprive())` in order to toggle

Reply:
Well that's the point, i don't want to toggle as a Client, just make it private. 
I keep the ToggleVisibility naming as we will use it further using password grant for a beneficiary to change it's own item visibility from the app